src/plugins/reporter-bugzilla.c

```diff
@@ -86,6 +86,24 @@ struct bugzilla_struct {
     GList       *b_private_groups;
 };
 
+static void set_default_settings(map_string_t *osinfo, map_string_t *settings)
+{
+    char *default_BugzillaURL;
+    parse_osinfo_for_bug_url(osinfo, &default_BugzillaURL);
+    /* if BugzillaURL is defined in conf_file or env , it will replace this value */
+    set_map_string_item_from_string(settings, "BugzillaURL", default_BugzillaURL);
+    log_debug("Loaded BUG_REPORT_URL '%s' from os-release", default_BugzillaURL);
+
+    char *default_Product;
+    char *default_ProductVersion;
```

Comment:
Good question.... my lack of recent `C` development is showing... my brain said "It went out of scope so don't worry."

I'll add the `free()` calls how I think they should look.

I'm not totally sure I'm calling `parse_osinfo_for_bz` with the right kind of arguments based on what you said regarding `set_default_settings` below.  I've been working in higher level languages for a while and it seems my familiarity with pointers is less good than I thought...